mods/beds/functions.lua

```diff
@@ -171,21 +175,26 @@ function beds.on_rightclick(pos, player)
 	end
 end
 
-
+----
 -- Callbacks
--- Only register respawn callback if respawn enabled
+----
+
+-- Only register respawn callback if respawn enabled.
 if enable_respawn then
-	-- respawn player at bed if enabled and valid position is found
+	-- Respawn player at bed if enabled and valid position is found.
 	minetest.register_on_respawnplayer(function(player)
 		local name = player:get_player_name()
-		local pos = beds.spawn[name]
-		if pos then
+		local pos = beds.spawn[name] -- Get the position of the player's bed.
```

Comment:
Too detailed comments. The function of these lines is not hard to understand. The comments take it way too far.